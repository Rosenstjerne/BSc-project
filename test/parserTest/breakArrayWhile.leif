var int a,b;
var int [] c;
c = new int[5];
c = [1,2,3,4,5];
a = 0;
b = 5;

while (a < b) {
    if (a == 3) {
        break;
    }
    print(c[a]);
    a = a + 1;
}

