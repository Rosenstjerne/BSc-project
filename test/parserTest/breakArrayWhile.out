function null main() {
	var int a, b;
	var int[] c;
	
	c = ((new int)[5]);
	a = ;
	b = 5;
	while((a < b)) {
	#begin while
		if((a == 3))
		#begin if
			break;
			
		#end if
		print((c[a]));
		a = (a + 1);
		
	}
	#end while
	
	
}
